src/config/dynamic-theme-fixes.config

```diff
@@ -12882,6 +12882,24 @@ body {
 
 ================================
 
+wiki.teamfortress.com
+
+INVERT
+.mw-wiki-logo
+
+CSS
+html, body {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+body, #mw-page-base, #p-logo, #content, #footer {
+    background-image: none !important;
+}
+div.vectorTabs ul li.selected *, #searchInput::placeholder {
```

Comment:
Not sure, but the current modified one by dark reader is in par with the theme of the site and should behave like these beige-ish colors.

Reply:
These elements are unreadable unless you change the color.

Reply:
```suggestion
div.vectorTabs ul li.selected a,
#searchInput::placeholder {
```